lmn/views/views_notes.py

```diff
@@ -44,3 +45,12 @@ def notes_for_show(request, show_pk):
 def note_detail(request, note_pk):
     note = get_object_or_404(Note, pk=note_pk)
     return render(request, 'lmn/notes/note_detail.html' , { 'note': note })
+
+# issue 5 delete note by chris
```

Comment:
Again this comment should be removed. Comments should explain how the code works

Reply:
@KelseyStiff @svenJonis138 removed the comment.